core/Updates/4.0.0-b1.php

```diff
@@ -210,6 +210,50 @@ public function doUpdate(Updater $updater)
             // switch to default provider if GeoIp Legacy was still in use
             LocationProvider::setCurrentProvider(LocationProvider\DefaultProvider::ID);
         }
+
+        // eg the case when not updating from most recent Matomo 3.X and when not using the UI updater
+        // afterwards the should receive a notification that the plugins are outdated
+        self::ensureCorePluginsThatWereMovedToMarketplaceCanBeUpdated();
```

Comment:
This logic runs for people that upgrade from 3.14.0 and older because there the `oneClickUpdatePartTwo` wouldn't be executed as it either didn't exist yet or it wasn't working. 
This logic runs also for people who update on the command line.

We then create the files during the update step so at least people will be made aware there's an update available on the marketplace.